chart/kubeapps/templates/_helpers.tpl

```diff
@@ -79,14 +79,14 @@ Create name for the chartsvc based on the fullname
 Create name for the dashboard based on the fullname
 */}}
 {{- define "kubeapps.dashboard.fullname" -}}
```

Comment:
the fact that the values and the templates are still called dashboard might be confusing, so we might want to change these too? What do you think?

Reply:
If we rename it to `kubeapps-dashboard-internal` the templates would still make sense. I wouldn't call it `react` to not make it depend on the underlying technology.